src/Normalizer/Normalizer.php

```diff
@@ -188,6 +213,15 @@ private function getLinksByLinkNormalizationMappings(
      */
     private function isWithinGroup(NormalizerContextInterface $context, $mapping): bool
     {
+        @trigger_error(
```

Comment:
this deprecation is not solveable by the user, should be only triggered if the groups are set

Reply:
move it further down, after the are groups empty check